### PR TITLE
FIX(SystemTests) Dynamic Log changes unable to check from kafka-pods

### PR DESCRIPTION
**Pull request by michalxo**
Signed-off-by: Michal Tóth <mtoth@redhat.com>

Tests were unable to connect to exposed Connect|MM2 service  and check for dynamically changed log level.
`io.strimzi.systemtest.log.LoggingChangeST#testDynamicallySetConnectLoggingLevels` checks this value using curl from kafkaClients pod.

`io.strimzi.systemtest.log.LoggingChangeST#testDynamicallySetMM2LoggingLevels` MM2 NetworkPolicy allows only connection from MM2 or cluster-operator pod -> thus this check is performed from MM2 pod

### CODE REVIEW

**Review by stanlyDoge — APPROVED** [review]

**Comment by michalxo** [comment]
@strimzi-ci run tests profile=regression testcase=LoggingChangeST#testDynamicallySetMM2LoggingLevels,LoggingChangeST#testDynamicallySetConnectLoggingLevels test_only

File: systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java [hunk 1]
```diff
@@ -848,13 +849,13 @@ void testDynamicallySetKafkaExternalLogging() {
 
         TestUtils.waitFor("Verify logger change", Constants.GLOBAL_POLL_INTERVAL, Constants.GLOBAL_TIMEOUT,
             () -> cmdKubeClient().execInPodContainer(KafkaResources.kafkaPodName(CLUSTER_NAME, 0),
-                        "kafka", "/bin/bash", "-c", "bin/kafka-configs.sh --bootstrap-server localhost:9092 --describe --entity-type broker-loggers --entity-name 0").out()
+                        "kafka-clients", "/bin/bash", "-c", "bin/kafka-configs.sh --bootstrap-server localhost:9092 --describe --entity-type broker-loggers --entity-name 0").out()
```
Comment (im-konge) [inline systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java:852]: Why is this changed? 
And `Kafka` doesn't have container `kafka-clients` if I'm right.

File: systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java [hunk 2]
```diff
@@ -864,6 +865,8 @@ void testDynamicallySetMM2LoggingLevels() throws InterruptedException {
         KafkaResource.kafkaEphemeral(CLUSTER_NAME + "-target", 3).done();
         KafkaClientsResource.deployKafkaClients(false, KAFKA_CLIENTS_NAME).done();
         KafkaMirrorMaker2Resource.kafkaMirrorMaker2(CLUSTER_NAME, CLUSTER_NAME + "-target", CLUSTER_NAME + "-source", 1, false).done();
+        String kafkaMM2PodName = kubeClient().listPodsByPrefixInName(CLUSTER_NAME + "-" + Constants.MIRROR_MAKER2).get(0).getMetadata().getName();
```
Comment (im-konge) [inline systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java:868]:
```suggestion
        String kafkaMM2PodName = KafkaMirrorMaker2Resource.kafkaMirrorMaker2Client().inNamespace(NAMESPACE).withName(CLUSTER_NAME).get().getMetadata().getName();
```

Comment (michalxo) [reply systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java:868]: Thanks, I will update it right away.

File: systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java (hunk 1 above)
Comment (michalxo) [reply systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java:852]: This is a mistake in completely different test. Thanks for spotting that!

File: systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java (hunk 2 above)
Comment (im-konge) [reply systemtest/src/test/java/io/strimzi/systemtest/log/LoggingChangeST.java:868]: Sorry, my bad :smile: I meant this:
```suggestion
        String kafkaMM2PodName = kubeClient().listPods(Labels.STRIMZI_KIND_LABEL, KafkaMirrorMaker2.RESOURCE_KIND).get(0).getMetadata().getName();
```

**Comment by michalxo** [comment]
@strimzi-ci run tests true profile=regression testcase=io.strimzi.systemtest.log.LoggingChangeST#testDynamicallySetMM2LoggingLevels

**Comment by im-konge** [comment]
@strimzi-ci run tests test_only profile=regression testcase=LoggingChangeST

**Review by Frawless — APPROVED** [review]

**Review by im-konge — APPROVED** [review]

**Review by scholzj — APPROVED** [review]
